### PR TITLE
docs: added fonts example

### PR DESCRIPTION
**Pull request by ggomaeng**
A lot of people have been asking in discord on using custom fonts for frog.

Improved the docs with some examples.

### CODE REVIEW

File: site/pages/concepts/images-intents.mdx [hunk 1]
```diff
@@ -129,6 +129,140 @@ export const app = new Frog({
...
+        >
+          Madimi One
+        </div>
+      </div>
+    ),
+  })
+})
+```
+
+```tsx [custom fonts]
```
Comment (dalechyn) [inline site/pages/concepts/images-intents.mdx]:
```suggestion
```tsx [Custom fonts]
```

File: site/pages/concepts/images-intents.mdx [hunk 2]
```diff
@@ -129,6 +129,140 @@ export const app = new Frog({
 
 [Read more on Image Options.](https://vercel.com/docs/functions/og-image-generation/og-image-api#fonts-parameters-within-options)
 
+### Fonts
+
+Below is an example of using `fonts` property in `imageOptions`.
+
+:::code-group
+
+```tsx twoslash [google fonts]
```
Comment (dalechyn) [inline site/pages/concepts/images-intents.mdx]:
```suggestion
```tsx twoslash [Google fonts]
```

File: site/pages/concepts/images-intents.mdx [hunk 3]
```diff
@@ -129,6 +129,140 @@ export const app = new Frog({
...
+  })
+})
+```
+
+```tsx [Custom fonts]
+import { Frog } from 'frog' 
+
+// from url
+import fetch from 'node-fetch';
+const urlFont = await fetch('https://...').then((res) => res.arrayBuffer());
```
Comment (jxom) [inline site/pages/concepts/images-intents.mdx]: We don't need to use `node-fetch`. Native is fine.

File: site/pages/concepts/images-intents.mdx [hunk 4]
```diff
@@ -129,6 +129,140 @@ export const app = new Frog({
...
+    ),
+  })
+})
+```
+
+```tsx [Custom fonts]
+import { Frog } from 'frog' 
+
+// from url
+import fetch from 'node-fetch';
```
Comment (dalechyn) [inline site/pages/concepts/images-intents.mdx]:
```suggestion
```

**Comment by madroneropaulo** [comment]
Any news on this? Fonts don't seem to be working for me. Using latest frog: (in this example I put one of the fonts hosted on our CDN)
```
const london = await fetch("https://assets.devpinata.cloud/OldLondon.ttf");
return c.res({
    image: <div style={{display: "flex", gap: 20}}>
      <p style={{fontFamily: "London", color: "white", fontSize: 60}}>
        Hello world
      </p>
      <p style={{color: "white", fontSize: 60}}>
        Hello world 2
      </p>
    </div>,
    imageOptions: {
      fonts: [
        {
          name: "London",
          data: await london.arrayBuffer(),
          weight: 600,
          style: "normal"
        }
      ]
    },
    intents: [],
  })
```

**Comment by dalechyn** [comment]
> Any news on this? Fonts don't seem to be working for me. Using latest frog: (in this example I put one of the fonts hosted on our CDN)


https://github.com/wevm/frog/issues/303#issue-2279191627

since this pr is not about the bug, please let's keep it out of this pr that adds docs on fonts.

**Review by tmm — APPROVED** [review]

**Pull request merged**